src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18200.cs

```diff
@@ -0,0 +1,29 @@
+# if TEST_FAILS_ON_ANDROID && TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_IOS  // It's a Windows specific API issue, so restricting the other platforms
```

Comment:
Because is a platform specific, here can use directly `#if WINDOWS`. Are not failing on other platforms, just have not sense.

Reply:
@jsuarezruiz , Thanks for pointing out. I have modified the changes